src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -515,10 +515,11 @@ network::~network() {
 //     [ executable primitive_inst ]
 //     [ memory reuse information ]
 void network::save(cldnn::BinaryOutputBuffer& ob) {
-    kernels_cache kernels_cache(get_engine(), _config, 0, nullptr, {""});
+    kernels_cache kernels_cache(get_engine(), _config, 0);
```

Comment:
BTW I recognized this was from the original implementation, but would like to ask just out of curiosity.
Can't we use program's kernels cache as is?

Reply:
I updated to reuse the program's kernels cache as you reviewed. Thank you.